vj4/app.py

```diff
@@ -33,13 +34,41 @@
                help='Expire time for changemail token, in seconds.')
 options.define('url_prefix', default='https://vijos.org', help='URL prefix.')
 options.define('cdn_prefix', default='/', help='CDN prefix.')
+options.define('sentry_dsn', default='', help='Sentry integration DSN.')
 
 _logger = logging.getLogger(__name__)
 
 
+class SentryMiddleware(aiohttp_sentry.SentryMiddleware): # For getting a correct IP
```

Comment:
For getting a correct client IP?

Reply:
The original class didnt consider the situation of using unix socket